spec/services/pii/attributes_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+describe Pii::Attributes do
+  describe '#new_from_hash' do
+    it 'initializes from plain Hash' do
+      pii = described_class.new_from_hash(first_name: 'Jane')
```

Comment:
I have been trying to use it to make it easier to rename classes, esp during early feature development.